### ISSUE TITLE
An API with allowed Coins

### ISSUE THREAD

**Issue by bifunctor** [khinkali/sink#29]
Hi @robertBrem 
We need an API with allowed Coins to provide for selection. 

Thanks

## Acceptance Criteria
```
    @Test(timeout = 2_000L)
    public void shouldReturnListOfCoins() throws IOException {
        String userId = System.getenv("APPLICATION_USER_ID");
        JsonArray coins = provider
                .target()
                .path("coins")
                .request()
                .header("Authorization", "Bearer " + getToken())
                .get(JsonArray.class);
        assertThat(toStringList(coins), hasItems("BTC", "ETH", "XRP"));
    }
```

**Comment by robertBrem** [comment]
implemented in `0.5.0`

**Comment by bifunctor** [comment]
Awesome work. Thanks
